controllers/upgrade_handlers.go

```diff
@@ -169,6 +174,14 @@ func (r *ImageBasedUpgradeReconciler) prePivot(ctx context.Context, ibu *lcav1al
 	return doNotRequeue(), nil
 }
 
+func (r *ImageBasedUpgradeReconciler) getDesiredStaterootRepo(ibu *lcav1alpha1.ImageBasedUpgrade) string {
+	return fmt.Sprintf("/host/ostree/deploy/rhcos_%s/var", ibu.Spec.SeedImageRef.Version)
```

Comment:
I don't think it is important, but we have defined functions like `PathOutsideChroot`, `GetStaterootPath` maybe they can be used here.

Also I don't think this function needs to be in reonciler.

Reply:
Added a todo to investigate it later and removed func from reonciler